lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -308,8 +314,15 @@ function cancelTerms(numerator, denominator) {
       return new CancelOutStatus(numerator, denominator, false);
     }
 
-    [numerator, newCoeff] = reduceStatus.newNode.args;
-
+    // Sometimes the fraction reduces to a constant e.g. 6 / 2 -> 3,
+    // in which case `newCoeff` (the denominator coefficient) should be null
```

Comment:
aw, sorry for missing this case in review :(

cancelling things is surprisingly tricky ._.

but it's nice that you have so many users that you can catch bugs fast :D I love that about the app